utils/misc.mli

```diff
@@ -257,6 +257,9 @@ val protect_writing_to_file
           channel. If the function raises an exception then [filename]
           will be removed. *)
 
+val concat_null_separated : string list -> string
+val split_null_separated : string -> string list
```

Comment:
A docstring is needed, especially explaining the fact that the result is NULL-terminated (which is different from the standard "concat" terminology in OCaml. In fact, a different name may be better: `concat_null_terminated`, `split_null_terminated`.

Reply:
changed and documented